Sources/SwiftDocC/Model/Rendering/RenderSectionTranslator/DeclarationsSectionTranslator.swift

```diff
@@ -59,7 +61,31 @@ struct DeclarationsSectionTranslator: RenderSectionTranslator {
                     )
                 )
             }
-        
+
+            if let alternateDeclarations = symbol.alternateDeclarationVariants[trait] {
+                for pair in alternateDeclarations {
+                    let (platforms, decls) = pair
+                    for alternateDeclaration in decls {
+                        let renderedTokens = alternateDeclaration.declarationFragments.map(translateFragment)
+
+                        let platformNames = platforms.sorted { (lhs, rhs) -> Bool in
+                            guard let lhsValue = lhs, let rhsValue = rhs else {
+                                return lhs == nil
+                            }
+                            return lhsValue.rawValue < rhsValue.rawValue
+                        }
```

Comment:
Can we reuse the sorted function defined on `Sources/SwiftDocC/Utility/FoundationExtensions/SortByKeyPath.swift`
```suggestion
		                let platformNames = platforms
		                    .compactMap { $0 }
		                    .sorted(by: \.rawValue)
```